CMakeLists.txt

```diff
@@ -342,6 +353,14 @@ if (WITH_DISNEY_MATERIAL)
     endif ()
 endif ()
 
+if (WITH_EMBREE)
+    if (NOT USE_EXTERNAL_EMBREE)
+        set (EMBREE_INCLUDE_DIRS ${APPLESEED_DEPS_STAGE_DIR}/include/)
```

Comment:
You're missing the Embree folder here.

Reply:
I prefer using #include <embree3/...> instead. I find this notation much more informative.

consider
```cpp
#include <embree3/rtcore.h>
```
and
```cpp
#include <rtcore.h>
```